docs/pages/guides/deep-linking.mdx

```diff
@@ -47,7 +47,7 @@ Build your native app with EAS Build to ensure the entitlement is registered wit
 
 <ConfigReactNative title="Manual native configuration">
 
-Apps that don't use [Continuous Native Generation](/workflow/continuous-native-generation) (`npx expo prebuild`) must [manually configure](/build-reference/ios-capabilities#manual-setup) the **Associated Domains** capability for their bundle identifier.
```

Comment:
This change does not look correct.

Reply:
you are right; I mistakenly made this change to the link